src/components/dialogs/csv-import-filter-creation.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import { useCSVReader } from 'react-papaparse';
+import Button from '@mui/material/Button';
+import React, { useState } from 'react';
+import Grid from '@mui/material/Grid';
+import { FormattedMessage, useIntl } from 'react-intl';
+import TextField from '@mui/material/TextField';
+import CsvDownloader from 'react-csv-downloader';
+import { createFilter } from '../../utils/rest-api';
+import { FilterType } from '../../utils/elementType';
+import { useSelector } from 'react-redux';
+import Alert from '@mui/material/Alert';
+import { equipmentsDefinition } from './generic-filter-dialog';
+import PropTypes from 'prop-types';
+
+const CsvImportFilterCreation = ({
```

Comment:
It's a dialog component then it should be named like others :
`CsvImportFilterCreationDialog`